### ISSUE TITLE
Updating broken links in readme file.

### ISSUE THREAD

**Issue by garimasingh128** [garimasingh128/gitsetgo#80]
There are several instances in the README where the URL of the project is incorrect. Correct it and replace it with https://github.com/garimasingh128/gitsetgo

**Comment by ritvij14** [comment]
Can I take this for CH20 or is it already taken?

**Comment by garimasingh128** [comment]
Someone already created a PR without getting assigned but I'll assign this to you.

**Comment by ritvij14** [comment]
Thanks a lot. I will make proper changes and send PR soon.

**Comment by ritvij14** [comment]
![Screenshot from 2020-09-12 12-44-31](https://user-images.githubusercontent.com/51456744/92989977-e4bd5680-f4f5-11ea-8880-8cc32b51a129.png)
![Screenshot from 2020-09-12 12-43-30](https://user-images.githubusercontent.com/51456744/92989978-e6871a00-f4f5-11ea-957d-aba4863a388a.png)

**Comment by ritvij14** [comment]
@garimasingh128 in the `hakincodes` branch, I did not find any instances of the links being wrong for this project as you mentioned.
The links for a couple of images were broken, which I have changed.

**Comment by garimasingh128** [comment]
Okay I will see to it.

**Comment by garimasingh128** [comment]
The README badges had the link to the other owner's repository before he transferred it. 
```
![Author](https://img.shields.io/badge/author-garimasingh128-orange)
![Author](https://img.shields.io/badge/author-mrinal41298-orange)
![License](https://img.shields.io/badge/license-MIT-brightgreen)
![Platform](https://img.shields.io/badge/platform-Visual%20Studio%20Code-blue)
![Maintained](https://img.shields.io/maintenance/yes/2020)
![Last Commit](https://img.shields.io/github/last-commit/mrinal41298/gitsetgo)
![Release Date](https://img.shields.io/github/release-date/mrinal41298/gitsetgo)
![Issues](https://img.shields.io/github/issues/mrinal41298/gitsetgo)
![Stars GitHub](https://img.shields.io/github/stars/mrinal41298/gitsetgo)
![Forks](https://img.shields.io/github/forks/garimasingh128/gitsetgo)
![Language](https://img.shields.io/github/languages/top/mrinal41298/gitsetgo)
![Size](https://img.shields.io/github/repo-size/mrinal41298/gitsetgo)
![Twitter](https://img.shields.io/twitter/url?url=https%3A%2F%2Fgithub.com%2Fgarimasingh128%2Fgitsetgo)
```

You need to fix these with the current link. The made with love badge also has a broken link.

**Comment by ritvij14** [comment]
@garimasingh128 this has been solved to by PR #82 so I think we can close this.
